README.md

```diff
@@ -1,289 +1,610 @@
-      kind: Binding
-      name: coligo-service-binding
-      id: coligo-service-binding
-  application:
-    name: nodejs-rest-http-crud
-    group: apps
-    version: v1
-    resource: deployments
+  name:                 # string, optional, default: .metadata.name
+  type:                 # string, optional
+  provider:             # string, optional
+
+  application:          # PodSpec-able resource ObjectReference
+    apiVersion:         # string
+    kind:               # string
+    name:               # string
+    containers:         # []intstr.IntOrString, optional
+    ...
+
+  service:              # Provisioned Service-able resource ObjectReference
+    apiVersion:         # string
+    kind:               # string
+    name:               # string
+    ...
+
+status:
+  conditions:           # []Condition containing at least one entry for `Ready`
+  - type:               # string
+    status:             # string
+    lastTransitionTime: # Time
+    reason:             # string
+    message:            # string
 ```
 
-#### Customizing data bindings
+## Example Resource
 
-The `ServiceBinding` CR has an element per service called `dataMappings`, whereby the author of the CR can rename certain binding items and/or compose of multiple items:
+```yaml
+apiVersion: service.binding/v1alpha1
+kind: ServiceBinding
+metadata:
+  name: online-banking-to-account-service
+spec:
+  name: account-service
 
-Partial sample of service-specific mappings:
-```
-  services:
-    - group: postgres.dev
-      kind: Service
-      name: global-user-db
-      version: v1beta1
-      id: postgres-global-user
-    - group: ibmcloud.ibm.com
-      version: v1alpha1
-      kind: Binding
-      name: watson-service-binding
-      id: watson-service-binding
-      dataMappings:
-        - name: WATSON_URL
-          value: {{  .status.host }} / {{ .status.port }}
-        - name: WATSON_USERNAME
-          value: {{  .status.username }}
+  application:
+    apiVersion: apps/v1
+    kind:       Deployment
+    name:       online-banking
+
+  service:
+    apiVersion: com.example/v1alpha1
+    kind:       AccountService
+    name:       prod-account-service
+
+status:
+  conditions:
+  - type:   Ready
+    status: True
 ```
 
-#### <kbd>EXPERIMENTAL</kbd> Synthetic data bindings
+## Reconciler Implementation
 
-If a `dataMappings` requires a cross-service composition then a new synthetic service entry must be created.  
+A Reconciler implementation for the `ServiceBinding` type is responsible for binding the Provisioned Service binding `Secret` into an Application.  The `Secret` referred to by `.status.binding.name` on the resource represented by `service` **MUST** be mounted as a volume on the resource represented by `application`.  If the `application` resource is managed by another Reconciler, a `ServiceBinding` Reconciler implementation **SHOULD** ensure that the `Secret` volume mount configuration remains after the other Reconciler completes.
```

Comment:
A reconciler alone in insufficient to handle `Job`, which is PodSpec-able, which is why we also run mutating webhooks for Knative bindings.  There have shared logic for the PodSpec transformation.

Reply:
I'll clean up the wording here so that implementations are not qualified as reconciler implementations.

Reply:
Addressed in 5ce7d50